libpdf/textbox.py

```diff
@@ -219,14 +219,20 @@ def render_chapters(  # pylint: disable=too-many-branches, too-many-locals
                     # The ghost chapter contains only the position from the outline and have no textbox
                     position = ghost_chapter_position_generator(chapter, chapter_page)
                     horizontal_box = None
+
+                    if 'virt.' in chapter['number']:
+                        LOG.info(
+                            'Virtual number %s is applied to chapter number, '
+                            'so this number may not be consistent with the numerical order in the content.',
+                            chapter['number'],
+                        )
+
                     LOG.info(
```

Comment:
This log message is only emitted in case 'ghost chapter' *and* 'virtual chapter'. Is this intended?